src/SingleCard.js

```diff
@@ -9,32 +9,46 @@ const useStyles = makeStyles((theme) => ({
   title: {
     color: 'white',
     background: 'blue',
-    fontFamily: 'HelsinkiGrotesk'
+    fontFamily: 'HelsinkiGrotesk',
+    fontSize: 24,
   },
   text: {
     fontSize: 16,
-    fontFamily: 'HelsinkiGrotesk'
+    fontFamily: 'HelsinkiGrotesk',
   },
   button: {
+    fontFamily: 'HelsinkiGrotesk',
     color: 'black',
     background: 'white',
+    fontSize: 16
   }
 }));
 
 function SingleCard(props) {
     const classes = useStyles();
-    const { title, text, button_text } = props;
+    const { title, text, button_text, button_url, width, height } = props;
 
-    return (
+    console.log('height' + height);
+    console.log('singlecard:' + width);
+    const goto = (url) => {
+      window.location.href = url;
+     }
+
+     return (
         <React.Fragment>
             <Card
               heading={title}
               className={classes.title}
+              style ={{
+                width: width ? width : '-1',
+                height: height ? height : '-1',
+              }}
               >
               <Typography dangerouslySetInnerHTML={{__html: text}} className={classes.text} />
               <Button
                 className={classes.button}
                 variant="secondary"
+                onClick={() => goto(button_url)}
               >
                 {button_text}
               </Button>
```

Comment:
A link here with 'href' would be better for accessibility.

SPA stuff could be used in the future with the react router. Maybe enough for now to have a real link.